oz/sqlalchemy/__init__.py

```diff
@@ -55,17 +57,21 @@ def engine():
         if oz.settings["db_pool_timeout"]:
             kwargs["pool_timeout"] = oz.settings["db_pool_timeout"]
 
-        _engine = create_engine(oz.settings["db"], **kwargs)
-        after_fork.registered = True
-        register_after_fork(after_fork, after_fork)
+        first_engine = len(_engines) == 0
+        _engines[connection_string] = create_engine(connection_string, **kwargs)
+
+        if first_engine:
```

Comment:
?? _engines is a `dict`, right? If so `if not _engines` is the shorthand for checking that `_engines` is an empty dictionary.